javascript/builtins/Error.json

```diff
@@ -372,7 +372,10 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": false
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": "12"
```

Comment:
We're actually dropping Edge Mobile from our compat tables.  See #3888 for more info.